community.html

```diff
@@ -369,15 +370,13 @@ <h2 class="section-header fw-600 mb-4">
                           <iframe
                             width="100%"
                             height="325"
-                            src="https://www.youtube.com/embed/tv3pyCLcJfQ"
+                            src="https://www.youtube.com/embed/8gbj9c_wb9Y"
                             title="YouTube video player"
                             frameborder="0"
                             allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
                             allowfullscreen
                           ></iframe>
-                          <p class="fw-600 mb-0 mt-2">
-                            OpenMetadata 0.12.0 Release
-                          </p>
+                          <p class="fw-600 mb-0 mt-2">October 27th, 2022</p>
```

Comment:
Not sure but can we use an array and loop over it to reduce change at multiple places? 
And others can update the data easily I feel

Reply:
Sure @chirag-madlani, We can improve the code in another patch.